index.html

```diff
@@ -70,7 +76,8 @@ <h2>MY RECENT <br> WORK</h2>
         <hr class="divider">    
         <section class="experience-container bg1">        
```

Comment:
- [ ] The hr divider here should cover the width of the whole project section as shown on the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10).